index.html

```diff
@@ -89,9 +89,7 @@ <h2 class="mdl-card__title-text card-headers">East Valley</h2>
             The original Coffee & Code based out of Heatsync Labs in Mesa.  This meetup was started in 2011 and is still going strong today!
           </div>
           <div class="mdl-card__actions mdl-card--border">
-            <a href="https://www.meetup.com/Coffee-and-Code-East-Valley/" class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect">
-              View Meetup
-            </a>
+            <a href="https://www.meetup.com/Coffee-and-Code-East-Valley/"><img src="images/if_meetup_2363057.png"></a>
```

Comment:
Looks like the ripple effect is removed from this, and it also will not match the styling of its fellow components.